src/UniqueQueue.php

```diff
@@ -85,15 +86,18 @@ public function addItems(array $items)
         try {
             $setName = $this->getSetName();
 
-            $pipe = $this->redis->pipeline();
-            foreach ($items as $item) {
-                $item = (string)$item;
-                if (empty($item)) {
-                    throw new Exception\InvalidArgument('Items in $items mustn\'t be empty');
+            foreach (array_chunk($items, self::OPT_ADD_MAX_CHUNK_SIZE) as $chunkItems) {
```

Comment:
... když nad tím tak přemýšlím, tak tohle integračně neotestuješ :/ nenapadá mě jak ... no každopádně tam máš chybu, musíš to brát z pole options